### ISSUE TITLE
fix names across languages

### ISSUE THREAD

**Issue by PolyglotOpenstreetmap** [TheFive/osmbc#758]
Hi,

I have another feature request, which might be hard to implement. I'm proofreading a lot, often across languages. I happens quite often that I find typos in names, usually just a single character. When I correct those (or anybody else for that matter), I would have to go and have a look at all the translations that already happened and correct it there too. Often I do this, and occasionally I forget. I think that to most of the other proofreaders, it occurs to them to go and have a look at the other translations.

So would it be possible, when somebody corrects a word, to look in the other languages for the original occurrence of that word and if it's there, propose to the user to correct it there as well? The tricky part is to figure out if a word is a name (especially in German). But since there would be a dialog with a question, the user can decide to replace it or not in the other languages.

**Comment by TheFive** [comment]
a) I assume, i skip the decision, wether something is a noun or a name.
b) second, do you do such changes "alone", so that the diff string only contains the change of one word ? than OSMBC can find out that the word Kristoph was changed to Christoph, and OSMBC can warn, if Kristoph is used in another language.
c) in more complex change situations, this can become more complex :-)

**Comment by TheFive** [comment]
https://osmbc.openstreetmap.de/changes/log/?user=Polyglot&blog=&table=&oid=&property=&date=2018-03-1

https://osmbc.openstreetmap.de/changes/269033 (this could work)
https://osmbc.openstreetmap.de/changes/268949 (would fit the requirement, but word will not appear in other language)
https://osmbc.openstreetmap.de/changes/268871 (would fit, but that is a language wise decision)
https://osmbc.openstreetmap.de/changes/268868 (would not fit and would not work)
https://osmbc.openstreetmap.de/changes/268866 (would possibly fit, and be ok)
https://osmbc.openstreetmap.de/changes/268863 (would may be fit, but not work)
https://osmbc.openstreetmap.de/changes/268860 (may be...)

**Comment by TheFive** [comment]
As long as i have not implemented something, you can use this list to have a look what you have done :-)

**Comment by PolyglotOpenstreetmap** [comment]
Like I said, it's a tough one. I was only thinking about the first example. I have changed bdiscroe 3 or 4 times to bdiscoe. Sint-Saint-Sankt-Maarten-Martin is indeed language dependent. 

If I understand correctly, it should be the only change done in one Save, for it to possibly work/be detected? I can live with that.

I removed Userio, because I think it should be Usuario, but I wasn't totally certain, so it seemed wiser to simply drop it, like it was in another article in the same issue.

The url where there is a space between harry and wood is another problem entirely... :-)
